File: Toph/C++/Formatted Numbers.cpp
//Formatted Numbers

#include <iostream>
using namespace std;
int main()
{
string Humu;
    getline(cin,Humu);
   int length=Humu.size();
   if(length<=3)
   {
       cout<<Humu<<endl;
   }
   else if(length==4)
   {
       cout<<Humu[0]<<","<<Humu[1]<<Humu[2]<<Humu[3];
   }
   else if(length==5)
   {
       cout<<Humu[0]<<Humu[1]<<","<<Humu[2]<<Humu[3]<<Humu[4];
   }
   else if(length==6)
   {
      cout<<Humu[0]<<Humu[1]<<Humu[2]<<","<<Humu[3]<<Humu[4]<<Humu[5];
   }
   else if(length==7)
   {
       cout<<Humu[0]<<","<<Humu[1]<<Humu[2]<<Humu[3]<<","<<Humu[4]<<Humu[5]<<Humu[6];
   }
   else if(length==8)
   {
       cout<<Humu[0]<<Humu[1]<<","<<Humu[2]<<Humu[3]<<Humu[4]<<","<<Humu[5]<<Humu[6]<<Humu[7];
   }
   else if(length==9)
   {
       cout<<Humu[0]<<Humu[1]<<Humu[2]<<","<<Humu[3]<<Humu[4]<<Humu[5]<<","<<Humu[6]<<Humu[7]<<Humu[8];
   }
    return 0;
}

File: Toph/Python/Formatted Numbers.py
# Formatted Numbers
user_input = input()
length = len(user_input)
if(length<=3):
    print(user_input)
elif(length==4):
    print(user_input[0]+","+user_input[1]+user_input[2]+user_input[3])
elif(length==5):
    print(user_input[0]+user_input[1]+",",user_input[2]+user_input[3]+user_input[4])
elif(length==6):
    print(user_input[0]+user_input[1]+user_input[2]+","+user_input[3]+user_input[4]+user_input[5])
elif(length==7):
    print(user_input[0]+","+user_input[1]+user_input[2]+user_input[3]+","+user_input[4]+user_input[5]+user_input[6])
elif(length==8):
    print(user_input[0]+user_input[1]+","+user_input[2]+user_input[3]+user_input[4]+","+user_input[5]+user_input[6]+user_input[7])
elif(length==9):
    print(user_input[0]+user_input[1]+user_input[2]+","+user_input[3]+user_input[4]+user_input[5]+","+user_input[6]+user_input[7]+user_input[8])
